deploy/docker-compose/.env.defaults

```diff
@@ -112,3 +112,4 @@ HOST_CLOUDSDK_CONFIG=
 # ---------------------------------------------------------
 
 OPS_CODE_BLOCK_MEMORY_LIMIT_IN_MB=256
+DISABLE_SLACK_INTERACTIONS=false
```

Comment:
You forgot the OPS prefix. It can join the slack connection variable a few lines above.